argus/environment.py

```diff
@@ -57,24 +58,64 @@ class BaseOpenstackEnvironmentPreparer(BaseEnvironmentPreparer):
     """
 
     def __init__(self, config_file, config_opts,
-                 start_commands, stop_commands):
+                 start_commands, stop_commands,
+                 list_services_commands, filter_services_regexes,

```

Comment:
The new parameters can be optional, since they aren't actually needed by all subclasses of EnvironmentPreparers. Basically, write them as default arguments ``list_services_commands=None`` etc